packages/devtools_app/lib/src/screens/performance/performance_screen.dart

```diff
@@ -51,7 +57,14 @@ class PerformanceScreen extends Screen {
 }
 
 class PerformanceScreenBody extends StatefulWidget {
-  const PerformanceScreenBody({super.key});
+  const PerformanceScreenBody({
+    super.key,
+    // This allows the body to modify the value that gets returned by the
+    // enclosing [PerformanceScreen]'s `showIsolateSelector` [ValueListenable].
```

Comment:
This makes me wonder generally whether this feature belongs in the Performance screen or the CPU profiler screen. Can you explain the user journey of how a user would use this feature to debug a performance problem, and what type of performance problem would the user be encountering?